src/main/java/com/mraof/minestuck/block/machine/HolopadBlock.java

```diff
@@ -34,21 +33,11 @@
 @MethodsReturnNonnullByDefault
 public class HolopadBlock extends MachineBlock implements EntityBlock
 {
-	public static final Map<Direction, VoxelShape> SHAPE = createRotatedShapes(2, 0, 1, 14, 6, 13);
-	public static final Map<Direction, VoxelShape> COLLISION_SHAPE;
-	//todo these are unused?
-	protected static final AABB HOLOPAD_TOP_AABB = new AABB(3/16F, 6/16F, 2.6/16F, 13/16F, 7/16F, 12.6/16F);
-	protected static final AABB HOLOPAD_CARDSLOT_AABB = new AABB(4/16F, 0F, 13.8/16F, 12/16F, 10.1/16F, 15.94/16F);
+	public static final Map<Direction, VoxelShape> SHAPE = new CustomVoxelShape(new double[]{2, 0, 1, 14, 6, 13}).createRotatedShapes();
```

Comment:
I believe the Holopad's Interaction Shape should match that of its Collision. Or at the very least add a box for the back part where the Captchalogue Card is supposed to be inserted.

Reply:
I've combined the two, and also made small further adjustments.